src/System.Management.Automation/resources/FileSystemProviderStrings.resx

```diff
@@ -243,6 +243,9 @@
   <data name="TargetCannotContainDeviceName" xml:space="preserve">
     <value>Cannot process path '{0}' because the target represents a reserved device name.</value>
   </data>
+  <data name="EncodingNotUsed" xml:space="preserve">
+    <value>Encoding not used when -AsByteStream specified.</value>
```

Comment:
Minor comment: maybe change to `... when '-AsByteStream' is specified`?

Reply:
done